### PR TITLE
adding an offset function to code.org/learn/local to prevent the map …

### PR DESCRIPTION
**Pull request by annaxuphoto**
…from overlaying the footer

Fix to the code.org/learn/local page so that the map does not cover the footer when scrolling down.

Before:
Screen Shot 2021-06-16 at 10 25 11 AM

After:
Screen Shot 2021-06-16 at 10 25 01 AM

## Links

- [jira](https://codedotorg.atlassian.net/browse/FND-1594?atlOrigin=eyJpIjoiMzFiMDUzYjU3NmYwNDFmMzgwMmMwMjcyYzMxNzBjNmMiLCJwIjoiaiJ9)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/sites/code.org/pages/public/learn/local.js [hunk 1]
```diff
@@ -52,6 +57,21 @@ $(function() {
     });
 });
 
+function checkOffset() {
```
Comment (daynew) [inline apps/src/sites/code.org/pages/public/learn/local.js:60]: Copying some documentation to here as well.
Also consider adding some inline comments below explaining the maths we are doing.
```suggestion
// Prevent the map from covering the footer
function checkOffset() {
```

File: apps/src/sites/code.org/pages/public/learn/local.js [hunk 2]
```diff
@@ -44,6 +44,11 @@ $(function() {
   // Make the map sticky.
   $('#map').sticky({topSpacing: 0});
 
+  //Prevent the map from covering the footer
```
Comment (daynew) [inline apps/src/sites/code.org/pages/public/learn/local.js]: nit
```suggestion
  // Prevent the map from covering the footer
```

**Review by daynew — APPROVED** [review]
Looks good. Just some minor comments.

**Review by daynew — APPROVED** [review]

**Comment by dju90** [comment]
Is it possible to have a little more white space between the map and the footer? I haven't pulled the branch to see what it looks like locally, but from the screenshot it looks like it bumps up right against the footer.

**Comment by annaxuphoto** [comment]
Yup, added a 20px space between the map bottom and footer

Screen Shot 2021-06-16 at 6 40 26 PM

**Pull request merged**